### ISSUE TITLE
[Supporting the Platform - Make Code - Extensions] Focus indicator is not visible on controls in high contrast theme under 'Extensions' screen.

### ISSUE THREAD

**Issue by sonimishra** [microsoft/pxt-microbit#2346]
**User Experience:**
User will find difficulty to see any control if keyboard focus is not visible on any interactive control in high contrast theme as they won't be able to take an action on such control and also they will surely miss the functionality and will proceed further.

**Test Environment:**
Version 1903 (OS Build 18867.1000)
Platform: Edge, Chrome

**Pre-Requisite:**
Turn on Windows High contrast theme (Settings->Ease of access->High Contrast->Turn on High Contrast->Select High Contrast White and Black

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/beta
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Extensions" option and select it.
7. Navigate through all the controls except 'learn more' link appearing under "Extensions" page
8. Verify focus visibility on controls like (Bluetooth services, Devices, Radio-broadcast, Servo etc.) after applying high contrast theme(Black/White).

**Actual Result:**
Keyboard focus is not visible on controls like (Bluetooth services, Devices, Radio-broadcast, Servo etc.) link after applying high contrast theme(Black/White) under 'Extensions' screen and due to this visual impaired user is unable to see the keyboard focus on that control and also he/she is unable to perform functionality over such control.
 
**Expected Result:**
Keyboard focus should be visible on controls like (Bluetooth services, Devices, Radio-broadcast, Servo etc.) link after applying high contrast theme(Black/White) under 'Extensions' screen,so that visual impaired user could easily see the keyboard focus on that control to perform functionality over such control.

**Note:** 
This issue also exists with Chrome.

**MAS Reference:**
[MAS 4.3.1 - No Disruption of Accessibility Features](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74})

[Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/PageNotFoundError.aspx?requestUrl=https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx
) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Comment by sonimishra** [comment]
[MAS4.3.1_extensions_HC.zip](https://github.com/microsoft/pxt-microbit/files/3471801/MAS4.3.1_extensions_HC.zip)

**Comment by abchatra** [comment]
Please re-test using beta site https://makecode.microbit.org/beta . If it reproduces let me know. I will reopen the bug.

**Comment by sonimishra** [comment]
Issue still exists in https://makecode.microbit.org/beta so reactivating the issue.

**Comment by jwunderl** [comment]
I followed the provided steps, and here's what I see (tested on mac, pc):

![2019-08-08 10 39 12](https://user-images.githubusercontent.com/5615930/62724948-f49c6a00-b9c8-11e9-8e79-f434d07bfc1f.gif)

Is this issue that the highlighting box in blue around the icon, and changing text color is unclear / not visible enough? The issue says not visible so I was expecting no highlighting

**Comment by sonimishra** [comment]
Issue has been verified and now the focus indicator is visible on the controls.
Hence closing the bug.
